core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminTitle() {
+    return t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminPreview() {
+    return t('Displays a news feed from BackdropCMS.org.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent() {
+    $feed = backdrop_http_request('https://backdropcms.org/api/backdrop-news');
+    $links = array();
+
+    if ($feed->code === '200') {
+      $news = json_decode($feed->data);
+      foreach ($news->nodes as $key => $node) {
+        $links[] = l(
+          $node->node->title,
+          $node->node->field_link
+        ) . ' - ' . $node->node->field_tagline;
+      }
+    }
+
+    $build = array(
+      '#theme' => 'dashboard_panel__news',
+    );
+    if (empty($links)) {
+      $build['list'] = array(
+        '#type' => 'markup',
+        '#markup' => t('No news at this time.'),
+      );
+    }
+    else {
+      $build['list'] = array(
+        '#theme' => 'item_list',
+        '#items' => $links,
+      );
+    }
+
+    return $build;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+
+    $form['url'] = array(
+      '#type' => 'help',
+      '#markup' => t(
+        'This block is providing a news feed from BackdropCMS.org and the contents can not be editted. You can remove it from the dashboard if you wish.'
```

Comment:
Typo: `editted`. Also, as @docwilmot pointed, people might use this block elsewhere, so we should remove the reference to the dashboard. Suggestion:

```suggestion
        'This block provides a news feed from BackdropCMS.org. Its contents can not be edited.'
```

...also, the contents of the `t()` function should not be wrapped onto a separate line.

Reply:
This string is not wrapped; the string itself is moved to a separate line; `'` are faster to parse for PHP and if there are not `'`s in the string itself the single quote wrapper is preferred in Backdrop.  

Thanks for spotting the typo.